### DISCUSSION TITLE
Not sure what it actually disables

### DISCUSSION THREAD

**Discussion by fredagsguf** [GriefPrevention/GriefPrevention#900] [Q&A] [answered]
Hi, what "features" does ```CheckPistonMovement``` disable when set to false?

I'm seeing a high usage from griefprevention regarding pistons, so i took a quick look and found that option - but i couldn't find anything on the wiki regarding what it disables other than the Piston movement event?

![image](https://user-images.githubusercontent.com/46964809/88317664-e5b5e100-cd19-11ea-8ab4-a10000b64fcd.png)

**Comment by Jikoo** [comment, marked as answer]
Exactly what it says on the tin, disables everything involving pistons.

In most worlds you probably won't see much of a difference in grief with or without it enabled, but it's certainly possible for someone to build a piston machine to push garbage in or pull blocks out of a claim so they can mine them. Was added in #671 if you want a more detailed read.

Key notes:

>The current workaround is to set LimitPistonsToLandClaims to true, but this prevents pistons from working in unclaimed areas.

> >With ignore piston movement off claims can be altered by non-owners from outside the claim using pistons.

>Yes, that's correct. It's a trade-off for the performance improvements.

**Comment by fredagsguf** [comment]
Ah okay, well i have coreprotect on my server so i think i will be fine.
Thank you for the quick answer :)

**Comment by RoboMWM** [comment]
Yea, got a couple config options that need to be documented. Will at least note this so we don't forget to add it whenever I decide on how to make the documentation wiki-like (allowing others to edit/submit edits) again.
